docs/reference/configuration.md

```diff
@@ -28,15 +28,43 @@ This reference uses `.` to denote the nesting of values.
 
 * `logging.format` `(one of ["json", "text"] : "text")` - Format to output log message in
 * `logging.level` `(one of ["TRACE", "DEBUG", "INFO", "WARN", "ERROR", "NONE"] : "DEBUG")` - Logging level to output
-* `logging.audit_log_level` `(one of ["TRACE", "DEBUG", "INFO", "WARN", "ERROR", "NONE"] : "DEBUG")` - Audit logs level to output. **Please notice that in case you configure this field to be lower than the main logger level, you won't be able to get the audit logs**
+* `logging.audit_log_level` `(one of ["TRACE", "DEBUG", "INFO", "WARN", "ERROR", "NONE"] : "DEBUG")` - Audit logs level to output.
+
+  **Note:** In case you configure this field to be lower than the main logger level, you won't be able to get the audit logs 
```

Comment:
the note will split lines - reference the name of the parameter or revert it back to the previous one

Reply:
This is how it's rendered, let me know if it's not clear enough:

![image](https://user-images.githubusercontent.com/3787958/184089132-8a40ce68-573c-42c8-b373-8331fd4d24eb.png)